### DISCUSSION TITLE
Cannot load types

### DISCUSSION THREAD

**Discussion by paola-guitian-papa** [dotansimha/graphql-code-generator#5252] [Q&A]
I am trying to set up graphql-code-generator for my React-Apollo-Client application. I set up graphl-code-generator using the initialization wizard. 

I am having trouble getting the paths  for my `documents` field. My understanding based on the docs is that "The documents field should point to your GraphQL documents: query, mutation, subscription and fragment". My file structure for my queries and mutations is as follows: 
src/utils/graphql/mutations/index
src/utils/graphql/queries/index

 therefore I set up my documents field to look like this: 
codegen.json:
```
{
  "overwrite": true,
  "schema": "https://private-endpoint/api/graphql",
  "documents": ["src/utils/graphql/queries/index", "src/utils/graphql/mutations/index"],
  "generates": {
    "src/utils/types/types.ts": {
      "plugins": [
        "typescript",
        "typescript-operations",
        "typescript-react-apollo"
      ]
    }
  }
}
```
 
However, I get the following error:

```
$ graphql-codegen --config codegen.json
  ✔ Parse configuration
  ❯ Generate outputs
    ❯ Generate src/utils/graphql
      ✔ Load GraphQL schemas
      ✖ Load GraphQL documents
        →           - src/utils/graphql/mutations/login
        Generate

 Found 1 error

  ✖ src/utils/graphql
    Error: 
          Unable to find any GraphQL type definitions for the following pointers:
        
              - src/utils/graphql/queries/account
              ,
              - src/utils/graphql/mutations/login
    

Something went wrong
error Command failed with exit code 1.

```

It seems to not find my gql definitions even though they are located in the files and the relative path is correct
Thanks!

**Comment by dotansimha** [comment]
Hi @paola-guitian-papa 
How does your `"src/utils/graphql/queries/index"` file looks like? is it a code file that exports operations?

**Reply by ardatan** [reply]
Maybe it is missing file extension because glob searches for complete path.

**Reply by paola-guitian-papa** [reply]
yes it is a file that exports the operations! Going to try that out
